portfolio/.classpath

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm, I would assume most of the Eclipse files that start with a "." shouldn't make it into the repo, since the repo normally shouldn't be opinionated about what IDE people should use.

But I've never used a production Java codebase before, so maybe @ssmall can chime in here.

Reply:
You're correct, these dotfiles should not be checked in

Reply:
Unresolving because this file is still included in the PR and should be removed

Reply:
Turns out a VS Code Java extension was generating this files upon deletion. Removed, files should stay out of the repo now.